src/chocolatey/infrastructure.app/domain/SourceTypes.cs

```diff
@@ -14,6 +14,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
+
 namespace chocolatey.infrastructure.app.domain
 {
     /// <summary>

```

Comment:
```suggestion
namespace chocolatey.infrastructure.app.domain
{

    using System;

    /// <summary>
```

Chocolatey convention is for using statements to go inside the namespace.

Reply:
Fixed!